p2p/debug/list.go

```diff
@@ -0,0 +1,133 @@
+package debug
```

Comment:
debug tag?

Reply:
adding `//go:build debug` causes the CLI to break, my guess is its not build with the debug tag? Will look into this further

Reply:
Doing a `ctrl f` for `go build` is a good starting point here.

<img width="575" alt="Screenshot 2023-07-25 at 3 47 51 PM" src="https://github.com/pokt-network/pocket/assets/1892194/0c96a69f-44fc-4164-b641-4018d06e2725">

Reply:
So the reason these cannot be put under debug is as follows. The cli calls the `PrintPeerList` function. So the `peer` subdir in the cli would also need to be with the debug tag. This calls the helper functions so that would need to be in a debug tag and thus all of the CLI would also need it.

I think it better to change the `debug` dir to something like `printing` or `introspect` perhaps.

WDYT @Olshansk